Drv/I2cDriverPorts/I2cStatusEnumAi.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Need to include this file in Drv/I2cDriverPorts/CMakeLists.txt as shown below:

```cmake
set(SOURCE_FILES
	"${CMAKE_CURRENT_LIST_DIR}/I2cPortAi.xml"
	"${CMAKE_CURRENT_LIST_DIR}/I2cStatusEnumAi.xml" # <----------- Add this 
)
```

Reply:
Thank you Saba, my mistake when copying over to my forked version. It's now fixed.